packages/trader/src/App/Components/Elements/TogglePositions/__tests__/toggle-positions-mobile.spec.jsx

```diff
@@ -0,0 +1,226 @@
+                barrier_count: 1,
+                bid_price: 9.96,
+                buy_price: 10,
+                contract_id: 214693307868,
+                contract_type: 'NOTOUCH',
+                is_sold: 0,
+                underlying: '1HZ100V',
+            },
+            contract_update: undefined,
+            details: 'test details',
+            display_name: 'Volatility 100 (1s) Index',
+            entry_spot: 2040.71,
+            id: 214693307868,
+            indicative: 9.96,
+            is_unsupported: false,
+            is_valid_to_sell: true,
+            payout: 22.69,
+            profit_loss: -0.04,
+            purchase: 10,
+            reference: 428397697088,
+            status: 'profit',
+            type: 'NOTOUCH',
+        },
+        {
+            barrier: 2048.16,
+            contract_info: {
+                account_id: 147849428,
+                barrier: '2048.16',
+                barrier_count: 1,
+                bid_price: 9.98,
+                buy_price: 10,
+                contract_id: 214693307999,
+                contract_type: 'NOTOUCH',
+                is_sold: 0,
+                underlying: '1HZ100V',
+            },
+            contract_update: undefined,
+            details: 'test details',
+            display_name: 'Volatility 100 (1s) Index',
+            entry_spot: 2048.01,
+            id: 214693307999,
+            indicative: 9.98,
+            is_unsupported: false,
+            is_valid_to_sell: true,
+            payout: 22.78,
+            profit_loss: -0.02,
+            purchase: 10,
+            reference: 428397697099,
+            status: 'profit',
+            type: 'NOTOUCH',
+        },
+    ],
+    currency: 'USD',
+    disableApp: jest.fn(),
+    enableApp: jest.fn(),
+    error: '',
+    is_empty: false,
+    onClickSell: jest.fn(),
+    onClickCancel: jest.fn(),
+    toggleUnsupportedContractModal: jest.fn(),
+};
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    NavLink: props => <div onClick={props.onClick}>Go to Reports</div>,
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Icon: () => <div>Test Icon</div>,
+}));
+
+jest.mock('../../EmptyPortfolioMessage', () => jest.fn(() => <div>Test Error</div>));
+
+jest.mock('App/Components/Elements/PositionsDrawer/positions-modal-card.jsx', () =>
+    jest.fn(() => <div>Position modal card</div>)
+);
+
+describe('<TogglePositionsMobile />', () => {
+    const mockProgressSliderStream = (mocked_store, mocked_props) => {
+        return (
+            <TraderProviders store={mocked_store}>
+                <TogglePositionsMobile {...mocked_props} />
+            </TraderProviders>
+        );
+    };
+
+    it('should render <TogglePositionsMobile /> with <TogglePositions /> and <Modal /> content if is_positions_drawer_on === true', () => {
+        const mock_root_store = mockStore({
```

Comment:
I defined `mock_root_store` in each test case as it was recommended in other PRs

Reply:
Sounds reasonable, thank you, I'll apply